core/shared/src/test/scala/cats/parse/LocationMapTest.scala

```diff
@@ -226,7 +226,7 @@ class LocationMapTest extends munit.ScalaCheckSuite {
         val lc = lm.toLineCol(offset)
 
         assertEquals(oc, Some(c))
-        assertEquals(lc, oc.map { case Caret(_, r, c) => (r, c) })
+        assertEquals(lc, oc.map { case c  => (c.line, c.col) })
```

Comment:
can we remove the `case` and the extra space here?

I think you want to run `fmt`.

Reply:
Yeah forgot to run formatting